app-editors/vis/vis-0.3.ebuild

```diff
@@ -3,13 +3,17 @@
 
 EAPI=5
 
+MY_PTV=0.1
+MY_PT=${PN}-test-${MY_PTV}
```

Comment:
Is there any reason to use `${PN}` here? It reduces readability, and is longer than plain `vis`…